app/src/main/java/com/example/taskyy/domain/usecases/RegisterUseCase.kt

```diff
@@ -1,14 +1,14 @@
 package com.example.taskyy.domain.usecases
 
-import com.example.taskyy.domain.objects.RegisterUser
+import com.example.taskyy.domain.data_transfer_objects.RegisterUserDTO
 import com.example.taskyy.domain.repository.AuthRepository
 import javax.inject.Inject
 
 class RegisterUseCase @Inject constructor(
     private val authRepository: AuthRepository
 ) {
 
-    fun registerUser(registerUser: RegisterUser) {
-        authRepository.registerUser(registerUser)
+    fun registerUser(registerUserDTO: RegisterUserDTO) {
+        authRepository.registerUser(registerUserDTO)
     }
```

Comment:
This was correct as it was before. In the last PR I mentioned that you'll need a domain model which you can use across the app and a data model (the DTO) you can only use in data which contains the JSON specification. This use case should only know the domain model therefore